### PR TITLE
fix: cash out fixes

### PR DESCRIPTION
**Pull request by mautematico**
fix: on and off should have a fixed width
fix: fist screen should be two p
fix: autofocus on fields after next
fix: spacing
fix: button position
fix: make the denominations a dropdown
fix: fiat should shown input suffix
fix: wizard should have splash image
fix: add missing warning icon
fix: eslint

### CODE REVIEW

**Comment by mautematico** [comment]
[Cash out fixes](https://trello.com/c/pzG3UMPP/142-cash-out-fixes)

File: new-lamassu-admin/src/pages/Cashout/WizardSplash.js [hunk 1]
```diff
@@ -2,29 +2,47 @@ import { makeStyles } from '@material-ui/core'
...
+    flex: 1,
+    paddingBottom: 141,
+    '& > div': {
+      paddingBottom: 72,
+      '& > h1': {
+        color: neon,
+        marginBottom: 12,
+        marginTop: 30,
+        textAlign: 'center',
+        '& > svg': {
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Cashout/WizardSplash.js]: There's no need for the child selectors, just create a new class and put the className on the element you want.

File: new-lamassu-admin/src/pages/Cashout/WizardStep.styles.js [hunk 2]
```diff
@@ -6,21 +6,58 @@ export default {
...
+  },
+  header: {
+    display: 'flex',
+    '& div': {
+      flex: 1,
+      paddingRight: 16
+    },
+    '& img': {
+      position: 'relative',
+      top: -32
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Cashout/WizardStep.styles.js]: No need for child selectors here either.

File: new-lamassu-admin/src/pages/Cashout/WizardStep.styles.js [hunk 3]
```diff
@@ -6,21 +6,58 @@ export default {
...
+  disclaimer: {
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    '& > p': {
+      '& > svg': {
+        float: 'left',
+        margin: [[-4, 16, 48, 0]]
+      }
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Cashout/WizardStep.styles.js]: nor here.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**